### PR TITLE
docs: add tables vs lists

### PR DESCRIPTION
**Pull request by dlimiter**
# Objective
Added new guidelines around use of tables versus lists.

# Motivation and Context 
To clarify usage and associated interaction patterns.

# Checklist
- [x] Source document reviewed by designers and engineers

# Branch preview
https://dev.cultureamp.design/dlimiter-tables-lists/guidelines/tables-lists/

### CODE REVIEW

File: site/docs/guidelines/tables-lists.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:4]: Just checking that you've deliberately left the summary paragraph blank? I wonder if you could pull the 'General' stuff up here. Maybe something like "Lists and tables can be quite visually similar and can sometimes represent the same data, but they have distinct patterns which shouldn't be mixed."

File: site/docs/guidelines/tables-lists.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## Need to know
+### Tables
+- Tables are for presenting information which gives the user the flexibility to sort the data to find their own patterns
+- Tables must include column headers for sorting
+
+### Lists
+- Lists are for presenting grouped information into scannable patterns. The content within a list can be stacked to create hierarchy within the data.
+- Lists are sortable by a ‘Sorted by’ Select, and do not have header labels
+- A single row of a list could make sense
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:20]: could or should make sense?

File: site/docs/guidelines/tables-lists.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### General
+- Tables and lists have distinct patterns which shouldn't be mixed
+- Lists and tables can be quite visually similar and can sometimes represent the same data. This guide will outline content and user considerations in determining which pattern is the most appropriate.  
+
+## Tables
+Tables display rows and columns of tabular data. 
+
+### For the user
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:29]: Check for consistent use of pronouns in these lists, e.g. 

- _Tables_ display rows... 
- _They_ help make...
- _Makes_ it efficient... (no pronoun)

File: site/docs/guidelines/tables-lists.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- They help make it easy to scan large amounts of data
+- Makes it efficient to find patterns in values
+- Data can be sorted by ascending or descending order using table column headers
+
+### For the designer
+- Tables must represent more than 1 row of tabular data, otherwise use a card to contain a small volume of data. (e.g. Tile / List)
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:35]: Link to (/components/tile/) ?

File: site/docs/guidelines/tables-lists.mdx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- For textual or numerical values, column cells can contain only 1 piece of information
+- Columns cells that require additional visual or interactive components (e.g. Avatar and name) can contain more than 1 piece of information:
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:37]: Link to (/components/avatar/) ?

File: site/docs/guidelines/tables-lists.mdx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  - Interactive components (such as buttons or selects)
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:38]: (/components/button/) + (/components/select/)

File: site/docs/guidelines/tables-lists.mdx [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+## Tables
+Tables display rows and columns of tabular data. 
+
+### For the user
+- They help make it easy to scan large amounts of data
+- Makes it efficient to find patterns in values
+- Data can be sorted by ascending or descending order using table column headers
+
+### For the designer
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:34]: I wonder if we can break up this list any further. There's a lot of bullet points here. Maybe subcategories could be: elements, structure, sorting... something like that.

File: site/docs/guidelines/tables-lists.mdx [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+- Tables must represent more than 1 row of tabular data, otherwise use a card to contain a small volume of data. (e.g. Tile / List)
+- For textual or numerical values, column cells can contain only 1 piece of information
+- Columns cells that require additional visual or interactive components (e.g. Avatar and name) can contain more than 1 piece of information:
+  - Interactive components (such as buttons or selects)
+  - Non-interactive elements with no natural grouping or order (such as avatars or icons)
+- Tabular data is structured into rows of cells of related information. Each row contains the same number of cells (although some of these cells may be empty), which provide values of properties of the thing described by the row. ([Source: W3C](https://www.w3.org/TR/tabular-data-model/#:~:text=Tabular%20data%20is%20data%20that,thing%20described%20by%20the%20row.))
+- A table allows the user to assess and control the data to find their own patterns
+- If there is an action it should be included to the far right of the table row
+- Tables can have interactive elements (E.g. checkboxes) to provide functions such as multi-select and bulk actions
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:43]: lower case e on e.g.

File: site/docs/guidelines/tables-lists.mdx [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+- Avatars can be considered as a data point and can be useful alongside a name to quickly scan for recognisable faces
+- Tables must be sortable via table column headers
+- By default, one column must be sorted (E.g. chronological order or magnitude). Always present the user with the most relevant data first.
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:46]: lower case e on e.g.

File: site/docs/guidelines/tables-lists.mdx [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+## Lists
+Lists group and stack information into logical and hierarchical patterns.
+
+### For the user
+- A good way to browse and surface information. E.g. Looking for names, viewing favourable scores
+
+### For the designer
+- Lists prioritise content. Metadata small, key information bigger
+- Lists are a collection of objects of the same type (E.g. A set of questions, or a group of people)
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:64]: lower case e on e.g.

File: site/docs/guidelines/tables-lists.mdx [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+- For rich data (e.g. Data visualization, avatars, hero data)
+- Surfacing the most interesting information is more important than finding detailed data
+- Lists can be either stacked horizontal rows of [Cards](/components/card) or gridded [Tiles](/components/tile)
+- Lists can also provide bulk actions
+- Lists must use a Select as a ‘sorting by’ function
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:69]: (/components/select/)

File: site/docs/guidelines/tables-lists.mdx [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+- **Lists also known as:** Cards, Tiles, Rows, Grids
+
+## Need to know
+### Tables
+- Tables are for presenting information which gives the user the flexibility to sort the data to find their own patterns
+- Tables must include column headers for sorting
+
+### Lists
+- Lists are for presenting grouped information into scannable patterns. The content within a list can be stacked to create hierarchy within the data.
+- Lists are sortable by a ‘Sorted by’ Select, and do not have header labels
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:19]: Link to (/components/select/)?

File: site/docs/guidelines/tables-lists.mdx [hunk 13]
```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
+tags: ["Data Tables", "Tabular Data", "Cards", "Tiles", "Rows", "Grids"]
+inComparingSection: true
+---
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:7]: Are we going to add visuals to this? Would be great to embed an example of a list and an example of a table here.

File: site/docs/guidelines/tables-lists.mdx (hunk 12 above)
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:19]: Maybe "column header labels" to make it clear we're not talking about the header header

File: site/docs/guidelines/tables-lists.mdx [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+### Tables
+- Tables are for presenting information which gives the user the flexibility to sort the data to find their own patterns
+- Tables must include column headers for sorting
+
+### Lists
+- Lists are for presenting grouped information into scannable patterns. The content within a list can be stacked to create hierarchy within the data.
+- Lists are sortable by a ‘Sorted by’ Select, and do not have header labels
+- A single row of a list could make sense
+
+### General
```
Comment (zakwash01) [inline site/docs/guidelines/tables-lists.mdx:22]: I would put this above the tables and list rules - either in the summary or immediately after the Need to know heading

**Review by zakwash01 — APPROVED** [review]
Looks good! Made a few suggestions.

File: site/docs/guidelines/tables-lists.mdx (hunk 1 above)
Comment (dlimiter) [reply site/docs/guidelines/tables-lists.mdx:4]: Love it 👍

File: site/docs/guidelines/tables-lists.mdx (hunk 13 above)
Comment (dlimiter) [reply site/docs/guidelines/tables-lists.mdx:7]: Yep, the embeds are tricky so future iteration

File: site/docs/guidelines/tables-lists.mdx (hunk 14 above)
Comment (dlimiter) [reply site/docs/guidelines/tables-lists.mdx:22]: Took it out completely - put one line in the summary, the other as a standalone bullet in Need to Know

File: site/docs/guidelines/tables-lists.mdx (hunk 7 above)
Comment (dlimiter) [reply site/docs/guidelines/tables-lists.mdx:32]: I've broken it up a bit, but we can iterate 👍

**Pull request merged**